### PR TITLE
activator: add page

### PR DESCRIPTION
**Pull request by sarav-egnaroinc**
created man page for activator command, a wrapper for sbt

----

- [x] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [ ] The page has 8 or fewer examples.

- [ ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [ ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/activator.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Run the sbt application with debug mode:
+
+`activator -jvm-debug 9999 run`
+
+- to list available feature/options:
+
+`activator help`
+
+- build a binary version of your sbt project to deploy :
```
Comment (pxgamer) [inline pages/common/activator.md]:
```suggestion
- Build a binary version of your sbt project to deploy:
```

File: pages/common/activator.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# activator
+
+> Activator aims to be a friendly one-stop-shop to bootstrap your Scala
```
Comment (pxgamer) [inline pages/common/activator.md]:
```suggestion
> Activator aims to be a friendly one-stop-shop to bootstrap your Scala.
```

File: pages/common/activator.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- compile an sbt project:
+
+`activator compile`
+
+- Run the sbt application with debug mode:
+
+`activator -jvm-debug 9999 run`
+
+- to list available feature/options:
```
Comment (pxgamer) [inline pages/common/activator.md]:
```suggestion
- List available features and options:
```

File: pages/common/activator.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# activator
+
+> Activator aims to be a friendly one-stop-shop to bootstrap your Scala
+
+- To start an sbt application:
+
+`activator run`
+
+- compile an sbt project:
```
Comment (pxgamer) [inline pages/common/activator.md]:
```suggestion
- Compile an sbt project:
```

File: pages/common/activator.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# activator
+
+> Activator aims to be a friendly one-stop-shop to bootstrap your Scala.
```
Comment (agnivade) [inline pages/common/activator.md]: No need to repeat "Activator" again. We can just say "A tool which".

"Scala" should be "Scala projects" ?

File: pages/common/activator.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- To start an sbt application:
+
+`activator run`
+
+- Compile an sbt project:
+
+`activator compile`
+
+- Run the sbt application with debug mode:
```
Comment (agnivade) [inline pages/common/activator.md]: end with  - "listening at a given port"

File: pages/common/activator.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+`activator -jvm-debug 9999 run`
```
Comment (agnivade) [inline pages/common/activator.md]: 9999 in tokens (`{{}}`)

File: pages/common/activator.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+`activator run`
+
+- Compile an sbt project:
+
+`activator compile`
+
+- Run the sbt application with debug mode listening at a given port:
+
+`activator -jvm-debug {{portno}} run`
```
Comment (agnivade) [inline pages/common/activator.md]: You can put back 9999 here. We prefer concrete values in tokens.

**Pull request closed without merging**